cg/models/orders/orderform_schema.py

```diff
@@ -0,0 +1,123 @@
+from enum import Enum
+from typing import List, Optional
+
+from cg.constants import DataDelivery, Pipeline
+from pydantic import BaseModel, constr
+
+
+class SexEnum(str, Enum):
+    male = "male"
+    female = "female"
+    other = "other"
+
+
+class PriorityEnum(str, Enum):
+    research = "research"
+    standard = "standard"
+    priority = "priority"
+    express = "express"
+    clinical_trials = "clinical trials"
+
+
+class ContainerEnum(str, Enum):
+    agilent_sureselect_cre = "Agilent Sureselect CRE"
+    agilent_sureselect_v5 = "Agilent Sureselect V5"
+    sureselect_focused_exome = "SureSelect Focused Exome"
+    twist_target_hg19_bed = "Twist_Target_hg19.bed"
+    other = "other"
+
+
+class StatusEnum(str, Enum):
+    affected = "affected"
+    unaffected = "unaffected"
+    unknown = "unknown"
+
+
+STATUS_OPTIONS = ()
+
+NAME_PATTERN = r"^[A-Za-z0-9-]*$"
+
+
+# Validating orderform
+class OrderSample(BaseModel):
+    age_at_sampling: Optional[str]
+    application: str
+    capture_kit: Optional[str]
+    case_id: str
+    cohorts: List[str] = None
+    comment: Optional[str]
+    concentration: int
+    concentration_sample: int
+    container: ContainerEnum = ContainerEnum.other
+    container_name: Optional[str]
+    customer: Optional[str]
+    custom_index: Optional[str]
+    data_analysis: Pipeline
+    data_delivery: DataDelivery
+    elution_buffer: Optional[str]
+    extraction_method: Optional[str]
+    family_name: Optional[str]
+    father: Optional[str]
+    formalin_fixation_time: Optional[int]
+    from_sample: Optional[str]
+    index: str
+    index_number: Optional[int]
+    index_sequence: Optional[str]
+    internal_id: Optional[str]
+    mother: Optional[str]
+    name: constr(regex=NAME_PATTERN)
```

Comment:
can the regexp be part of the model? Since its never modified as of now

Reply:
Looks like pydantic models does not support meta variables. The NAME_PATTERN is expected to be reused in other fields in next pr